x-pack/plugins/observability_solution/logs_explorer/common/constants.ts

```diff
@@ -74,3 +74,20 @@ export const SMART_FALLBACK_FIELDS = {
 // UI preferences
 export const DEFAULT_COLUMNS = [RESOURCE_FIELD_CONFIGURATION, CONTENT_FIELD_CONFIGURATION];
 export const DEFAULT_ROWS_PER_PAGE = 100;
+
+// Content column filter options
+export const FILTER_FOR_CONTENT = [
```

Comment:
**nit**: it took me a second more looking at the usage of this list to understand that we want to filter OUT by these prefixes, maybe we could rename this to something more descriptive like `FILTER_OUT_FIELDS_PREFIXES` or similar, wdyt?